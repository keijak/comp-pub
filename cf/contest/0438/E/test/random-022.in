187 147
310
271
806
588
974
270
917
178
844
687
47
276
161
727
87
321
288
416
610
246
446
910
478
316
258
128
471
465
59
119
265
182
660
120
731
185
628
989
81
239
718
716
916
413
642
335
601
905
950
694
682
412
518
850
760
217
420
647
214
805
333
976
532
562
788
683
65
890
340
347
966
372
68
843
428
189
504
580
767
833
550
305
509
85
89
573
394
640
575
389
611
799
22
153
776
560
964
676
405
521
274
629
208
937
887
878
354
171
632
307
400
449
38
768
547
215
94
225
362
830
155
30
691
650
969
841
175
54
812
678
388
879
895
758
28
255
499
15
552
836
130
762
206
695
817
474
183
751
401
83
52
166
429
868
749
259
616
290
858
226
223
24
626
824
764
529
475
496
789
191
984
546
983
652
635
418
61
558
928
835
778
774
963
612
109
525
93
