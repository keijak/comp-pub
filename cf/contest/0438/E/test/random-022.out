0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
1
0
1
0
0
0
1
0
3
0
0
0
0
0
0
4
1
4
0
0
0
4
2
9
4
1
2
0
4
0
24
4
20
0
2
0
19
16
35
31
8
15
0
31
9
120
35
94
15
19
20
86
137
147
227
49
118
0
221
91
629
267
477
177
144
233
437
982
753
1453
376
821
130
1408
909
3294
2132
2514
1692
958
2181
2384
6745
4360
9091
2972
5565
1970
8809
7907
17921
16080
14259
13856
6820
17584
15076
44271
28926
55814
24729
36728
22133
54468
64706
101345
117444
86380
105852
51545
132271
106428
285548
206343
344036
202816
243146
209348
344249
500566
607227
831006
565445
771163
415833
949752
